AAC Events/AAC Events/AboutViewController.swift

```diff
@@ -60,43 +61,128 @@ class AboutViewController: UIViewController, SideMenuItemContent {
+        
+        runningDescAttrString.append(NSAttributedString(string: "\n\nOUR CORE VALUES", attributes: boldAttributes as [NSAttributedStringKey : Any]))
+        
+        runningDescAttrString.append(NSAttributedString(string: "\n\n• Build a sense of community around a diverse Afghan-American identity, while learning and embracing Afghan culture and history. \n• Provide a space for networks built on professional and academic connections for guidance, career growth, and development of leadership skills.\n• Engage in critical dialogue around challenging issues and activate the change to find solutions in our community.\n• Internalize norms and values that demonstrate respect, accountability, and flexibility towards different perspectives.  ", attributes: descriptionAttributes as [NSAttributedStringKey : Any]))
+        
+        runningDescAttrString.append(NSAttributedString(string: "\n\nA WAY FORWARD", attributes: boldAttributes as [NSAttributedStringKey : Any]))
+        
+        runningDescAttrString.append(NSAttributedString(string: "\n\nThis is where the Afghan-American Conference comes into the picture. AAC provides a forum for our community to address these difficult topics in a respectful, forward-thinking manner. It is a grassroots effort at shedding light on these pressing issues, inspiring action, and activating change from within the community. We won’t be able to address everything, nor do we have the answers to almost anything. Let us get the conversation started, let us share resources, and let us find those solutions together as a community.\n\n", attributes: descriptionAttributes as [NSAttributedStringKey : Any]))
 			
 		return runningDescAttrString
 		
 
 	}
 	
-	func getTheConferenceText() -> NSAttributedString {
+	func getOurCommittment() -> NSAttributedString {
 		let boldAttributes = [NSAttributedStringKey.font: boldFont]
 		let descriptionAttributes = [NSAttributedStringKey.font: descriptionFont]
         
-        let runningConfAttrString = NSMutableAttributedString()
+        let diveristyAttributes = [
+            NSAttributedStringKey.link: URL(string: "https://www.afghanamericanconference.org/diversity-1")!,
```

Comment:
These links are gonna be a problem. If they change them the app needs a new release to fix it. You should add them to dropbox and create a new endpoint that can just be a list of the URLS.

Reply:
I think for this AAC its fine if the links are just in the app.